README.md

```diff
@@ -1 +1,68 @@
-# sui-tealium-tracker
+# @s-ui/tealium-tracker
+> Tool to track events like click or change and, if a tag is present on the element dispatch a link event to tealium.
+
+This tool is meant to be used with the [@s-ui/html-tagger](https://github.com/SUI-Components/sui/tree/master/packages/sui-html-tagger) tool but you can define your own tags and it will work asswell.ls
+cd
+# Use:
+
+## Basic usage: 
+
+To use it require it from the desired file as this:
+```javascript
+require('@s-ui/tealium-tracker')
```

Comment:
That is fine ?!

I think that the corect one is `require('@s-ui/tealium-tracker')()`

Reply:
Typo error  on the readme, sorry going to change it

Reply:
Not a big fan of requiring and self executing it, as our code is isomorphic and it will enforce you do to a require inline the code. I rather prefer import it and execute if needed.